CONTRIBUTING.md

```diff
@@ -38,15 +38,15 @@ Only start working on the Pull Request after the team assigned the issue to you
 
 The following rule governs code contributions:
 
-* Contributions must be licensed under the [Apache 2.0 License](LICENSE)
+* Contributions must be licensed under the [Apache 2.0 License](/LICENSES/Apache-2.0.txt)
```

Comment:
If we keep the LICENSE file for Github, it might be confusing to link the one in the LICENSES folder

Reply:
you are right, I will change these links